gtk-doc.make adds $(srcdir) to $(HTML_IMAGES) for us.

--- origsrc/gst-plugins-bad-0.10.13/docs/plugins/Makefile.am	2009-06-05 14:00:28.000000000 -0500
+++ src/gst-plugins-bad-0.10.13/docs/plugins/Makefile.am	2009-07-28 01:10:10.848762800 -0500
@@ -159,7 +159,7 @@ EXTRA_HFILES = \
 	$(top_srcdir)/sys/dvb/gstdvbsrc.h
 
 # Images to copy into HTML directory.
-HTML_IMAGES = $(srcdir)/figures/*.png
+HTML_IMAGES = figures/*.png
 
 # Extra SGML files that are included by $(DOC_MAIN_SGML_FILE).
 content_files =
